packages/web/src/components/apps/workspace/preview.tsx

```diff
@@ -4,27 +4,38 @@ import { useEffect, useState } from 'react';
 import { Loader2Icon } from 'lucide-react';
 import { useLogs } from '../use-logs';
 import { Button } from '@srcbook/components/src/components/ui/button';
+import { usePackageJson } from '../use-package-json';
 
 type PropsType = {
   isActive?: boolean;
+  onShowPackagesPanel: () => void;
   className?: string;
 };
```

Comment:
I don't beleive this `onShowPackagesPanel` prop is used anymore - I meant to get rid of it. I'll do that on monday, or feel free to do it for me.